yaml/boundaries.yaml

```diff
@@ -131,29 +138,18 @@ filters:
   #    `unrecognized_*` (where XX is the viewpoint / country code and * is the
   #    kind of boundary it is).
   #
-  - filter: {admin_level: '2', boundary: claim, claimed_by: true, geom_type: line}
-    min_zoom: 8
-    output:
-      <<: *output_properties
-      kind: unrecognized_country
-      claimed_by: {col: claimed_by}
-      recognized_by: {col: recognized_by}
-    table: osm
-  - filter: {admin_level: '4', boundary: claim, claimed_by: true, geom_type: line}
-    min_zoom: 8
-    output:
-      <<: *output_properties
-      kind: unrecognized_region
-      claimed_by: {col: claimed_by}
-      recognized_by: {col: recognized_by}
-    table: osm
-  - filter: {admin_level: '6', boundary: claim, claimed_by: true, geom_type: line}
+  - filter:
+      all:
+        - geom_type: line
+        - admin_level: ['2', '4', '6']
+        - boundary: claim
+        - any:
+            - claimed_by: true
+            - disputed_by: true
```

Comment:
Should `recognized_by: true` be in this list for completeness?

Reply:
ah yes - I left it out because I don't understand why someone would be recognizing something if no one was claiming it.  But I think that there are cases where the claimant doesn't have an ISO code?

Reply:
fixed in 9506b97782500824588c5745c04dc520b202da17